chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -324,14 +335,6 @@
   end
 else
   if node["platform_version"].to_f < 12.0
-    cookbook_file "/etc/init.d/crowbar" do
-      owner "root"
-      group "root"
-      mode "0755"
-      action :create
-      source "crowbar.suse"
-    end
-
```

Comment:
I'd just drop the whole `if node["platform_version"].to_f < 12.0` block since it makes no sense anymore without an init script.